docs/how-tos/working-with-postgresql-data-types.rst

```diff
@@ -0,0 +1,121 @@
+.. _working_with_types:
+
+Working with PostgreSQL data types
+==================================
+
+PostgREST makes use of PostgreSQL text representation to work with data types. Thanks to this, you can use special literal values, such as ``now`` for timestamps, ``yes`` for booleans or time values including the time zones. This page describes how you can use literal values for different PostgreSQL types.
```

Comment:
Btw, TIL about 'yes', 'on' for booleans:

https://www.postgresql.org/docs/current/datatype-boolean.html

That page mentions the "string representation" keyword, perhaps we should use that one instead of "text representation"? The latter could be confusing since there's already a "text" type.

Reply:
So like:

```suggestion
PostgREST makes use of PostgreSQL string representations to work with data types. Thanks to this, you can use special values, such as ``now`` for timestamps, ``yes`` for booleans or time values including the time zones. This page describes how you can take advantage of these string representations to perform operations on different PostgreSQL data types.
```